src/libraries/Microsoft.Extensions.Http/src/Microsoft.Extensions.Http.csproj

```diff
@@ -33,7 +33,7 @@ System.Net.Http.IHttpClientFactory</PackageDescription>
     <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Extensions.Options\src\Microsoft.Extensions.Options.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
```

Comment:
Please prefer `TFI == .NETCoreApp` when `NetCoreAppMinimum` is equal or greater than the min version in the condition. The MSBuild functions aren't for free.